### PR TITLE
Add v1.16.1 images

### PR DESCRIPTION
**Pull request by ashie**

### CODE REVIEW

File: docker-image/v1.16/debian-cloudwatch/Gemfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+gem "fluent-plugin-grok-parser", "~> 2.6.2"
+gem "fluent-plugin-prometheus", "~> 2.0.2"
+gem 'fluent-plugin-json-in-json-2', ">= 1.0.2"
+gem "fluent-plugin-record-modifier", "~> 2.1.0"
+gem "fluent-plugin-detect-exceptions", "~> 0.0.13"
+gem "fluent-plugin-rewrite-tag-filter", "~> 2.4.0"
+gem "fluent-plugin-parser-cri", "~> 0.1.0"
+gem "aws-sdk-cloudwatchlogs", "~> 1.0"
+gem "fluent-plugin-cloudwatch-logs", "~> 0.14.0"
+gem "fluent-plugin-kubernetes_metadata_filter", "~> 3.2.0"
```
[the remark this answers is not in the document]
Comment (ashie) [reply docker-image/v1.16/debian-cloudwatch/Gemfile:20]: No particular reason.
Just forgot to update.

**Comment by ashie** [comment]
I've added elasticsearch8 images in this release, elasticsearch6 will be dropped.

File: docker-image/v1.16/debian-azureblob/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# AUTOMATICALLY GENERATED
+# DO NOT EDIT THIS FILE DIRECTLY, USE /templates/Dockerfile.erb
+
+FROM fluent/fluentd:v1.16.1-debian-amd64-1.2
```
Comment (kenhys) [inline docker-image/v1.16/debian-azureblob/Dockerfile:4]: Missing image.

```suggestion
FROM fluent/fluentd:v1.16.1-debian-amd64-1.0
```

Comment (kenhys) [reply docker-image/v1.16/debian-azureblob/Dockerfile:4]: templates/Dockerfile.erb is not touched.

Comment (ashie) [reply docker-image/v1.16/debian-azureblob/Dockerfile]: Done.

**Review by kenhys — APPROVED** [review]
LGTM

**Comment by ashie** [comment]
Thanks for your review!

**Pull request merged**